### DISCUSSION TITLE
antd表格超过500行数据，勾选比较卡顿，目前有解决方案吗？

### DISCUSSION THREAD

**Discussion by strongyc** [ant-design/ant-design#34524] [General]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
大数据表格勾选

### What does the proposed API look like?
能够解决1000行左右表格勾选不卡顿

**Comment by alienzhangyw** [comment]
对Table进行二次封装，用map来记录selectedRows（Map<key, record>）,自己重写onRow和rowSelection.onChange的逻辑，只把selectedRowKeys传给antd Table就行了。另外可以用虚拟列表优化性能。
